internal/app/app.go

```diff
@@ -36,7 +36,7 @@ func New() *App {
 	if cfg.DataBaseDSN == "" {
 		ns = primitivestorage.NewStorage(fh, err)
 	} else {
-		ns, err = dbstorage.NewDB(cfg.DataBaseDSN, ctx)
+		ns, err = dbstorage.NewDB(cfg.DataBaseDSN, &ctx)
```

Comment:
```suggestion
		ns, err = dbstorage.NewDB(ctx, cfg.DataBaseDSN)
```
Контекст принято передавать первым элементом, и передается он не по адресу, а напрямую. Если посмотреть в код, то функция Background() возвращает интерфейс. Интерфейс - ссылочный тип данных в Go.

Reply:
+